### ISSUE TITLE
"Buy goods" button

### ISSUE THREAD

**Issue by David-Ro** [bettermtl/bettermtl.github.io#39]
one more button option to click on: buy book. Or, buy material.
E.g. author selling book

**Comment by pcuci** [comment]
Can be done. What's a better abstraction that would work for most events
that have collateral for sale?

**Comment by David-Ro** [comment]
Buy what needs buying 
"Buy what needs buying"
"Achetez ce qui doit s'acheter"
or just
"What needs buying"
"Ce qui doit s'acheter"
So, there's a button to say "I'd like to show my interest" and another button that shows "I'm pretty much ready to buy now so get me started in the buying process"

**Comment by pcuci** [comment]
I need this defined before implementing it, which one do we go for? See Jaime's 3 buttons at the bottom of the activity: http://bettermtl.github.io/event/tlmux-jaime-levy-what-is-ux-strategy.html
- More
- Purchase colateral
- Register

**Comment by David-Ro** [comment]
I'd delete the More button. As it is, it takes people away, over to a
source website.

On Sun, Apr 10, 2016 at 8:36 PM, Paul Cuciureanu notifications@github.com
wrote:

> I need this defined before implementing it, which one do we go for? See
> Jaime's 3 buttons at the bottom of the activity:
> http://bettermtl.github.io/event/tlmux-jaime-levy-what-is-ux-strategy.html
> - More
> - Purchase colateral
> - Register
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> https://github.com/bettermtl/bettermtl.github.io/issues/39#issuecomment-208101986

## 

David

**Comment by robertpragai** [comment]
I agree with David, you want to keep people on the site and the information is very similar to the information already on betterMTL.
For a "More" section, it could be a link within the site to a bio of the person. This could be a section in which highlighted people are displayed, such as a "speaker corner" which is a section that shows all the people who are active in the scene, what they have accomplished and what they are working on.

For links which take you to other websites, they should always open a new tab/

**Comment by pcuci** [comment]
> For links which take you to other websites, they should always open a new

That's not best practice

You can middle-click or Ctrl+click to open a new tab/window. Users know
this, expected behaviour. We're not trying to keep people on our site - we
want them to go to events and socialise themselves. Shortest path to
registration.

Later, they'll be able to RSVP directly from bettermtl

**Comment by David-Ro** [comment]
must allow anyone to wiki-edit, to add stuff worth buying: this makes it an ecommerce microservice! @pcuci @robertpragai
